### DISCUSSION TITLE
Getting this error: Failed to load resource: the server responded with a status of 403 ()

### DISCUSSION THREAD

**Discussion by Shohrat-Husain** [vercel/next.js#49701] [App Router]
I am receiving the following error: 'Failed to load resource: the server responded with a status of 403 ()'. When I deployed my project on Bitbucket, the routing is not working properly. When I refresh the page, it redirects to the homepage, but the URL does not change.

**Comment by volpster2100** [comment]
If it’s a 403, sounds like a permission/access problem.
